tests/garage/torch/misc/test_algo_utils.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for utilities in torch/algos/_utils.py."""
+import torch
+
+from garage.torch.algos import compute_advantages
+
+
+class TestAlgoUtils:
```

Comment:
just an FYI that pytest doesn't require you define a class for this (you can just use functions called `test_...`)